source/databricks/package/file_writers/process_step_result_writer.py

```diff
@@ -188,11 +202,27 @@ def _write_result_to_table(
                 f"Unsupported aggregation_level, {aggregation_level.value}"
             )
 
-        (
-            df.write.format("delta")
-            .mode("append")
-            .option("mergeSchema", "false")
-            .option("overwriteSchema", "false")
-            .option("path", self.__delta_table_path)
-            .saveAsTable(self.__table_name)
+        df = (
+            df.withColumn(Colname.batch_id, lit(self.__batch_id))
+            .withColumn(Colname.batch_process_type, lit(self.__batch_process_type))
+            .withColumn(
+                Colname.batch_execution_time_start,
+                lit(self.__batch_execution_time_start),
+            )
         )
+
+        df = df.withColumnRenamed(
+            PartitionKeyName.GROUPING, "aggregation_level"
+        )  # TODO: rename Grouping enum to AggragationLevel
+        df = df.withColumnRenamed(
+            Colname.quality, "quantity_quality"
+        )  # TODO: use QuantityQuality everywhere
+        df = df.withColumnRenamed("quarter_time", "time")  # TODO: time everywhere
```

Comment:
```suggestion
        df = df.withColumnRenamed("quarter_time", Colname.quarter_time)
```
or event better: refactor our code so this rename is not needed.

Reply:
I can change it to Colname.time? I guess there is no reason to call it quarter_time...especially if we introduce a resolution column also. But again, I prefer not to do too many things in this PR